api/v1alpha1/eacruntime_types.go

```diff
@@ -0,0 +1,218 @@
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Enabled or Disabled for the components.
+	// Default enable.
+	// +optional
+	Disabled bool `json:"disabled,omitempty"`
+
+	// NodeSelector is a selector which must be true for the master to fit on a node.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// Whether to use host network or not.
+	// +kubebuilder:validation:Enum=HostNetwork;"";ContainerNetwork
+	// +optional
+	NetworkMode NetworkMode `json:"networkMode,omitempty"`
+}
+
+// EACFuseSpec is a description of the EAC Fuse
+type EACFuseSpec struct {
+	// The version information that instructs fluid to orchestrate a particular version of EAC Fuse
+	Version VersionSpec `json:"version,omitempty"`
+
+	// Configurable properties for EAC fuse
+	// +optional
+	Properties map[string]string `json:"properties,omitempty"`
+
+	// Resources that will be requested by EAC Fuse. <br>
+	// <br>
+	// Resources are not allowed for ephemeral containers. Ephemeral containers use spare resources
+	// already allocated to the pod.
+	// +optional
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// If the fuse client should be deployed in global mode,
+	// otherwise the affinity should be considered
+	// +optional
+	Global bool `json:"global,omitempty"`
+
+	// NodeSelector is a selector which must be true for the fuse client to fit on a node,
+	// this option only effect when global is enabled
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// CleanPolicy decides when to clean EAC Fuse pods.
+	// Currently Fluid supports two policies: OnDemand and OnRuntimeDeleted
+	// OnDemand cleans fuse pod once th fuse pod on some node is not needed
+	// OnRuntimeDeleted cleans fuse pod only when the cache runtime is deleted
+	// Defaults to OnRuntimeDeleted
+	// +optional
+	CleanPolicy FuseCleanPolicy `json:"cleanPolicy,omitempty"`
+
+	// Whether to use hostnetwork or not
+	// +kubebuilder:validation:Enum=HostNetwork;"";ContainerNetwork
+	// +optional
+	NetworkMode NetworkMode `json:"networkMode,omitempty"`
+}
+
+// EACRuntimeSpec defines the desired state of EACRuntime
+type EACRuntimeSpec struct {
+	// The component spec of EAC master
+	Master EACCompTemplateSpec `json:"master,omitempty"`
+
+	// The component spec of EAC worker
+	Worker EACCompTemplateSpec `json:"worker,omitempty"`
+
+	// The spec of init alifuse
+	InitAlifuse InitAlifuseSpec `json:"initAlifuse,omitempty"`
+
+	// The component spec of EAC Fuse
+	Fuse EACFuseSpec `json:"fuse,omitempty"`
+
+	// Tiered storage used by EAC worker
+	TieredStore TieredStore `json:"tieredstore,omitempty"`
+
+	// The replicas of the worker, need to be specified
+	Replicas int32 `json:"replicas,omitempty"`
+
+	// Enable OS other than AliyunOS
+	// Non-AliyunOS is not enabled by default
+	// +optional
+	EnableNoneAliyunOS bool `json:"enableNoneAliyunOS,omitempty"`
+
+	// Aliyun AccessKey ID for DirQuota API
```

Comment:
I don't suggest using AccessKeyID here. Instead, you can using the encrypted  option in the dataset.